### ISSUE TITLE
false translation in french of "run" in the "edit python fixes", here run mean "exécutez" and not "courses"

### ISSUE THREAD

**Issue by chcoste** [GoldenCheetah/GoldenCheetah#4526]
Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

**Comment by amtriathlon** [comment]
Hi, if you want to collaborate to fix the French translation we have some guidelines in https://github.com/GoldenCheetah/GoldenCheetah/wiki/Guidelines-for-translators and there is a proposed update pending of review in  https://groups.google.com/g/golden-cheetah-users/c/KVwt8LDsWLY/m/V5ARYu7aAAAJ, that same thread is a better place to propose changes too. Thank you.
